pyop2/base.py

```diff
@@ -436,6 +447,14 @@ def _is_indirect_and_not_read(self):
         return self._is_indirect and self._access is not READ
 
     @property
+    def _is_readonly(self):
+        return self._access == READ
+
+    @property
+    def _is_written(self):
```

Comment:
@FabioLuporini: @kynan was referring to the name of the property, not its existence/absence.  I.e. if you have `_is_readonly` then you would expect this to be called `_is_writeable` rather than `_is_written`.